tests/app/questionnaire/routing/test_when_rule_evaluator.py

```diff
@@ -49,6 +49,7 @@ def get_when_rule_evaluator(
     return WhenRuleEvaluator(
```

Comment:
This file has tests for each of the value source, so a new one should be added for `response_metadata`.